test/test_ensemble_hydrostatic_free_surface_models.jl

```diff
@@ -37,6 +39,62 @@ const CAVD = ConvectiveAdjustmentVerticalDiffusivity
     @test all(sic_model.velocities.u.data[1, 1, :] .≈ per_model.velocities.u.data[1, 1, :])
     @test all(sic_model.velocities.v.data[1, 1, :] .≈ per_model.velocities.v.data[1, 1, :])
     @test all(sic_model.tracers.c.data[1, 1, :]    .≈ per_model.tracers.c.data[1, 1, :])
+
+    @info "Testing forcing a single column grid model with a FieldTimeSeries..."
```

Comment:
it looks like this tests both single column and periodic

Reply:
Another way to do this test would be to add a single column grid test alongside wherever `update_model_field_time_series` is already tested. That might be less test code / easier to maintain.